h/views/api/groups.py

```diff
@@ -137,9 +138,6 @@ def upsert(context, request):
 
     Otherwise, replace the existing group's resource properties entirely and update
     the object.
-
-    :arg context:
-    :type context: h.traversal.GroupUpsertContext
```

Comment:
Outdated type documentation: `GroupUpsertContext` was renamed to `GroupContext` long ago